src/GraphsOptim.jl

```diff
@@ -1,11 +1,11 @@
 """
-    GraphsOptim
+	GraphsOptim
 
 A package for graph optimization algorithms that rely on mathematical programming.
 """
 module GraphsOptim
 
-using Graphs: AbstractGraph, is_directed
+using Graphs: AbstractGraph, is_directed, DiGraph
```

Comment:
why is this needed?

Reply:
Removed, use the same algorithm for undirected graphs.